packages/notebook-extension/schema/tracker.json

```diff
@@ -511,11 +511,21 @@
       "keys": ["ArrowUp"],
       "selector": ".jp-Notebook:focus"
     },
+    {
+      "command": "notebook:move-cursor-up",
+      "keys": ["ArrowUp"],
+      "selector": ".jp-Notebook-footer"
+    },
     {
       "command": "notebook:move-cursor-up",
       "keys": ["K"],
       "selector": ".jp-Notebook:focus"
     },
+    {
+      "command": "notebook:move-cursor-up",
+      "keys": ["K"],
+      "selector": ".jp-Notebook-footer"
```

Comment:
Can we collapse collapse this with the entry above by using a selector like `.jp-Notebook-footer, .jp-Notebook:focus`?

Reply:
Discussion keeps going in https://github.com/jupyterlab/jupyterlab/pull/14796#discussion_r1320825853.

Reply:
Unfortunately no (at least not with the ", " syntax); see https://github.com/jupyterlab/jupyterlab/pull/14796#discussion_r1320825853